src/components/Navbar.tsx

```
@@ -1,22 +1,38 @@
-                </nav>
-            </div>
-            <div className={styles["fade-navbar-effect"]}></div>
-            <div className={styles["empty-navbar"]}></div>
-        </>
+        <div className={styles["navbar-container"]}>
+            <nav className={styles["navbar"]}>
+                <Link href="/" className={styles["brand"]}>
+                    <h3 className={styles["title"]}>Old School Game</h3>
+                </Link>
+
+                <button className={styles["menu-icon"]} onClick={toggleMenu} aria-label="Toggle menu">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)